toolbox/include/base/BlockInformation.h

```diff
@@ -59,19 +60,22 @@ class wbt::BlockInformation {
 public:
     virtual ~BlockInformation();
 
-    //Block Options methods
+    // Block Options methods
+    // =====================
 
     /**
      * Convert a block option from its Toolbox identifier to a specific implementation
      *
-     * @param [in] key identifier of the block option
-     * @param [out] option implementation-specific block option
-     * @return true if the option has been converted. False otherwise
+     * @param [in] key    identifier of the block option
+     * @param [in] option implementation-specific block option
```

Comment:
Is this an in or out parameter?

Reply:
An out parameter, I don't even remember why I changed this